calico/getting-started/kubernetes/managed-public-cloud/aks.md

```diff
@@ -9,15 +9,91 @@ Enable {{site.prodname}} in AKS managed Kubernetes service.
 
 ### Value
 
-AKS has built-in support for {{site.prodname}}, providing a robust implementation of the full Kubernetes Network Policy API. AKS users wanting to go beyond Kubernetes network policy capabilities can make full use of the {{site.prodname}} Network Policy API. 
+AKS has built-in support for {{site.prodname}}, providing a robust implementation of the full Kubernetes Network Policy API. AKS users wanting to go beyond Kubernetes network policy capabilities can make full use of the {{site.prodname}} Network Policy API.
+
+You can also use {{site.prodname}} for networking on AKS in place of the default Azure VPC networking. This allows you to take advantage of the full set of {{site.prodname}} networking features.
 
 ### How to
 
-To enable {{site.prodname}} network policy enforcement, follow these step-by-step instructions: {% include open-new-window.html text='Create an AKS cluster and enable network policy' url='https://docs.microsoft.com/en-us/azure/aks/use-network-policies' %}.
+#### Install AKS with {{site.prodname}} for network policy
 
 The geeky details of what you get:
 {% include geek-details.html details='Policy:Calico,IPAM:Azure,CNI:Azure,Overlay:No,Routing:VPC Native,Datastore:Kubernetes' %}
 
+To enable {{site.prodname}} network policy enforcement, follow these step-by-step instructions: {% include open-new-window.html text='Create an AKS cluster and enable network policy' url='https://docs.microsoft.com/en-us/azure/aks/use-network-policies' %}.
+
+#### Install AKS with {{site.prodname}} networking
+
+The geeky details of what you get:
+{% include geek-details.html details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:VXLAN,Routing:Calico,Datastore:Kubernetes' %}
+
+1. Create an Azure AKS cluster with no Kubernetes CNI pre-installed. Please refer to [Bring your own CNI with AKS](https://docs.microsoft.com/en-us/azure/aks/use-byo-cni?tabs=azure-cli) for details.
+
+  ```bash
+  # Install aks-preview extension
+  az extension add --name aks-preview
+  # Update aks-preview to ensure latest version is installed
+  az extension update --name aks-preview
+
+  # Create a resource group
+  az group create --name $RESOURCE_GROUP_NAME --location $LOCATION
+
+  az aks create --resource-group $RESOURCE_GROUP_NAME --name my-calico-cluster --location $LOCATION --network-plugin none
+
+  ```
+
+1. Now that you have a cluster configured, you can install {{site.prodname}}.
```

Comment:
I'm finding that having run the commands above, I don't have a kubeconfig for the cluster.  Do we need another command here to get the kubeconfig?

Reply:
```suggestion
1. Get credentials to allow you to access the cluster:
    ```
    az aks get-credentials --resource-group $RESOURCE_GROUP_NAME  --name my-calico-cluster
    ```

1. Now that you have a cluster configured, you can install {{site.prodname}}.
```

Reply:
Yes. We need this command.